data/args.yml

```diff
@@ -4,6 +4,9 @@ version: "1.3"
 # The full Istio version identifier the docs describe
 full_version: "1.3.0"
 
+# The previous Istio version identifier the docs describe, used for upgrade documentation
+previous_version: "1.2"
```

Comment:
In order to reference this symbol from markdown files, you need to add a shortcode definition in layouts/shortcodes.

Reply:
You should also update the release instructions in README.md to indicate this field needs to be updated when creating a new release.